kmp-library/src/commonMain/kotlin/com/novoda/aws/v4/signer/HttpRequest.kt

```diff
@@ -0,0 +1,40 @@
+/*
+  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
```

Comment:
Should we remove this javadoc?

Reply:
482dbad963ecb0cbaab811a38f218d7dc5a7e7c1